jsAgent/main.js

```
@@ -0,0 +1,178 @@
+            }
+        } catch (e) {
+            console.log(e);
+        }
+    },
+    bindLive: function (selector, event, cb, cnx) {
+        bw.util.addEvent(cnx || document, event, function (e) {
+            var qs = (cnx || document).querySelectorAll(selector);
+            if (qs) {
+                var el = e.target || e.srcElement,
+                    index = -1;
+                while (el && (index = Array.prototype.indexOf.call(qs, el)) === -1)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)